x-pack/plugins/osquery/cypress/tasks/navigation.ts

```diff
@@ -47,7 +47,7 @@ export const NEW_FEATURES_TOUR_STORAGE_KEYS = {
   TIMELINES: 'securitySolution.security.timelineFlyoutHeader.saveTimelineTour',
   TIMELINE: 'securitySolution.timeline.newFeaturesTour.v8.12',
   FLYOUT: 'securitySolution.documentDetails.newFeaturesTour.v8.14',
-  ATTACK_DISCOVERY: 'securitySolution.attackDiscovery.newFeaturesTour.v8.14',
+  KNOWLEDGE_BASE: 'elasticAssistant.knowledgeBase.newFeaturesTour.v8.16',
```

Comment:
I don't _think_ this tour will interfere with your tests since I have the `const isTestAutomation =  window.Cypress != null;` logic in the tour component now, but it does not hurt to have it here.